src/components/components/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,113 @@
+.HeroSection {
+  padding-top: 5%;
+  max-width: 1260px;
+  width: 80%;
+  max-height: 673px;
+  height: 80vh;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 3%;
+
+  &__info {
+    padding: 30px 0;
+
+    &__title {
+      margin: 0;
+      font-size: calc(32px + (56 - 32) * ((100vw - 768px) / (1280 - 768)));
+      line-height: 1.33em;
+      color: var(--color-bunting);
+
+      &--circled {
+        padding: 0 10px;
+        border: 3px solid var(--color-blue-ribbon);
+        border-radius: 30px;
+      }
+    }
+
+    &__text {
+      font-size: calc(16px + 2 * (100vw / 1280));
+      color: #7d7cac;
```

Comment:
Please use the custom property here instead

Reply:
you mean I should add it to the global styles?

Reply:
yes, please!